extensions/amp-story-interactive/0.1/amp-story-interactive-abstract.js

```diff
@@ -333,7 +336,16 @@ export class AmpStoryInteractive extends AMP.BaseElement {
         while (options.length < optionNumber) {
           options.push({'optionIndex': options.length});
         }
-        options[optionNumber - 1][splitParts.slice(2).join('')] = attr.value;
+        const key = splitParts.slice(2).join('');
+        if (key === 'image') {
+          options[optionNumber - 1][key] = maybeMakeProxyUrl(
+            attr.value,
+            this.element,
```

Comment:
We want to pass in `this.element.getAmpDoc()` or `this.getAmpDoc()` (probably the latter)